src/client/components/Meals/Meals.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import ReactStars from "react-rating-stars-component";
+import '@mobiscroll/react-lite/dist/css/mobiscroll.min.css';
+import Pics from '../Helper/Pics';
+
+const Meals = (props) => {
+    const [pics, setPics] = useState(Pics)
```

Comment:
No need to be a state, you're not changing it anyways

Reply:
Changed